multimodal/src/autogluon/multimodal/utils/matcher.py

```diff
@@ -501,3 +502,7 @@ def convert_data_for_ranking(
     response_data = pd.DataFrame({response_column: data[response_column].unique().tolist()})
 
     return data_with_label, query_data, response_data, label_column
+
+
+def is_matching(pipeline: str):
```

Comment:
Shall we use https://github.com/autogluon/autogluon/blob/b7a144aefcd90b4ac6f110fb9d20f73c6de1af26/multimodal/src/autogluon/multimodal/predictor.py#L176 ?

```python
problem_property_dict[pipeline].is_matching
```

Reply:
There is circular import issue if importing `problem_property_dict ` from `predictor.py`. We may refactor this later.